guides/common/modules/proc_installing-the-discovery-service.adoc

```diff
@@ -13,15 +13,16 @@ The Discovery service requires a Discovery image, which is provided with {Projec
 The Discovery image uses a minimal operating system that is booted on hosts to acquire initial hardware information and check in with {Project}.
 
 ifdef::satellite[]
-The Foreman Discovery image provided with {Project} is based on {EL} 8.
+The Foreman Discovery image provided with {Project} is based on {EL} 9.
 endif::[]
 ifdef::orcharhino[]
-The Foreman Discovery image provided with {Project} is based on CentOS Stream 8.
+The Foreman Discovery image provided with {Project} is based on CentOS Stream 9.
```

Comment:
@maximiliankolb Can you please confirm that orcharhino FDI has been updated as well?